### ISSUE TITLE
accuracy_score wrong order 

### ISSUE THREAD

**Issue by unnir** [nadbordrozd/text-top-model#1]
you've made a mistake in your code, which affects the final scores. 

shortly, you have place y_true on the wrong place, here:

https://github.com/nadbordrozd/text-top-model/blob/d3099a39e90255c5c3942bf3e29516b7fbca3e3a/ttm/benchmarks.py#L37

check the docs:
http://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html

**Comment by nadbor-aimia** [comment]
I agree that it's a mistake, but a harmless one because `accuracy_score(preds, y_test)` is the same as `accuracy_score(y_test, preds)` - check [the implementation](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/metrics/classification.py)

**Comment by unnir** [comment]
hah you are right, but if and only if you have the binary problem ;)

**Comment by nadbor-aimia** [comment]
No, works like this for binary or multclass classification and even for multilabel classification. As well it should, accuracy is just a fraction of records for which `prediction == actual` and this is the same as `actual == prediction`

**Comment by unnir** [comment]
you right again 👍  sorry for opining the issue

**Comment by nadbor-aimia** [comment]
No worries. It's still a mistake.
